.github/workflows/ios-android-test.yml

```diff
@@ -43,6 +43,10 @@ jobs:
                   cache: 'gradle'
             - name: Test Android
               uses: ReactiveCircus/android-emulator-runner@76c2bf6f95ed6458fd659a1bdb680a0f8df232dc
+              if: |
+                !github.head_ref
+                && ( startsWith(github.ref, 'refs/heads/release') == true
+                     || startsWith(github.ref, 'refs/heads/main') == true )
```

Comment:
Nit: No need for the `startsWith` here, because we only mean _exactly_ `main`, whereas with `release` it could also be `release-$hotfixversion`.